checkov/sast/checks/base_check.py

```diff
@@ -0,0 +1,12 @@
+from checkov.common.models.enums import CheckCategories
+
+
+class BaseSastCheck():
+    def __init__(self, name: str, id: str, severity = None) -> None:
+        self.name = name
+        self.id = id
+        self.categories = [CheckCategories.SAST]
+        # TODO
```

Comment:
is this todo still needed? if so maybe expand what should be done?

Reply:
We need to fill it in the future when we will have the final SAST checks format